README.md

```diff
@@ -1,31 +1,59 @@
+
 # INTRO TO RAILS
-This is an Intro to Rails course, intended for eventual inclusion in the Girl Develop It Core Curriculum. All material by Cheri Allen, inspired by Railsbridge.
+
+Slides and materials are hosted at [https://github.com/girldevelopit/](https://github.com/girldevelopit/).
+
+## Course description
+
+This is an Intro to Rails course, intended for eventual inclusion in the Girl Develop It Core Curriculum. All material inspired by Railsbridge.
 
 The course is meant to be taught in four two-hour workshops. Each of the slides and practice files are customizable according to the needs of a given class or audience.
 
-Cheri's slides can be viewed [here](http://cherimarie.github.io/gdi-core-rails/#/).
+**Cost:** 
 
-## Promotional Blurb
-Ready to build on your basic Ruby knowledge to start building full web applications? It's time to learn Rails! 
+**Dates:** 
 
-Rails is a framework that makes creating web applications relatively quick and easy. There is a thriving community around it, a strong job market, and ample resources available to beginners. Rails powers applications like Living Social, Groupon, Twitter, Square, Github and more.
+**Location:** 
```

Comment:
This came from the [slides template](https://github.com/girldevelopit/gdi-slides-template)